rln/types.go

```diff
@@ -62,15 +62,32 @@ type RateLimitProof struct {
 	RLNIdentifier RLNIdentifier `json:"rlnIdentifier"`
 }
 
+type TreeDepth int
+
+const (
+	TreeDepth20 TreeDepth = 20
```

Comment:
Why these specific depths? Would be great to know the reasoning.

Reply:
These are the available tree depths that are embedded in zerokit's RLN: https://github.com/vacp2p/zerokit/tree/master/rln/resources